trello_service.py

```diff
@@ -0,0 +1,98 @@
+from trello_cards import Item
+import os
+import json
+import requests 
+
+TRELLO_SERVICE_ENDPOINT='http://api.trello.com/1/'
+def get_auth_params():
+    return {    
+                'key': os.environ.get('TRELLO_KEY'), 
+                'token': os.environ.get('TRELLO_TOKEN') 
+            }
+
+def service_url_get_boards():
+    return TRELLO_SERVICE_ENDPOINT+'/members/me/boards'
+
+def service_get_list_from_board(boardID):
+    return TRELLO_SERVICE_ENDPOINT+'/boards/'+boardID+'/lists'
+
+def service_create_card():
+     return TRELLO_SERVICE_ENDPOINT+'/cards'
+
+def service_get_cards_of_list(listId):
+     return TRELLO_SERVICE_ENDPOINT+'/lists/'+listId+'/cards'
+
+def service_move_card_url(cardId):
+     return (TRELLO_SERVICE_ENDPOINT+'/cards/%s' % cardId)
+
+
+
+def get_all_boards():
+    response = requests.request("GET", service_url_get_boards(), params=get_auth_params())
+    boards = response.text
+    return boards
+
+def get_board_by_name(name):
```

Comment:
This is unused (and by extension, get_all_boards is effectively unused) - you are using a TRELLO_BOARD_ID env var instead of finding one by name.

Don't merge unused code.